plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinition.tsx

```diff
@@ -159,8 +158,9 @@ export const OpenApiDefinition = ({
       <SwaggerUI
         spec={def}
         url=""
-        requestInterceptor={requestInterceptor}
         deepLinking
+        oauth2RedirectUrl={`${window.location.protocol}//${window.location.host}/oauth2-redirect.html`}
```

Comment:
Also I don't think all people want this default?

Reply:
It is the default defined in the documentation of the plugin. 
Currently it is not working (cf [issue](https://github.com/backstage/backstage/issues/11478)), I just applied the fix to have this working like the documentation.
But anyway, now anybody can override it